2-mandatory/1-writers.js

```diff
@@ -61,7 +61,12 @@ Exercise 1:
 */
 function logAllWriters() {
   // write your code to log all writers here
-};
+  for (let writer in writers) {
+    console.log(
+      `Hi, my name is ${writers[writer].firstName} ${writers[writer].lastName}. I am ${writers[writer].age} years old, and work as a ${writers[writer].occupation}.`
```

Comment:
good job! But I think  ${writer.firstName} is easier than ${writers[writer].firstName}.